packages/components/organisms/f-checkout/src/store/checkout.module.js

```diff
@@ -214,6 +245,10 @@ export default {
                 ...state.customer,
                 ...customer
             };
+        },
+
+        UPDATE_BASKET_DETAILS (state, { serviceType }) {
```

Comment:
Shouldn't these follow the same syntax setup as the other mutations? i.e the `:` & `=>` are missing here and in a few other places. I think it still fine without tho so happy to leave as is for now, might be one for making consistent later.

Reply:
Fixed